docs are good, but still shellcheck issues
```
dev-support/bin/create-release:540:1: note: The mentioned syntax error was in this brace group. [SC1009]
dev-support/bin/create-release:585:3: error: Couldn't parse this simple command. Fix to allow more checks. [SC1073]
dev-support/bin/create-release:586:5: error: Place shellcheck directives before commands, not after. [SC1126]
dev-support/bin/create-release:586:32: error:  Fix any mentioned problems and try again. [SC1072]
```

--- a/dev-support/bin/create-release
+++ b/dev-support/bin/create-release
@@ -581,9 +581,8 @@ function makearelease
   fi
 
   # Create SRC and BIN tarballs for release,
-  # shellcheck disable=SC2046
+  # shellcheck disable=SC2046,SC2086
   run_and_redirect "${LOGDIR}/mvn_${target}.log" \
-    # shellcheck disable=SC2086
     "${MVN}" ${MVN_ARGS[*]} ${target} \
       -Pdist,src,yarn-ui \
       "${signflags[@]}" \
@@ -613,7 +612,7 @@ function makearelease
   # we need to do install again so that jdiff and
   # a few other things get registered in the maven
   # universe correctly
-  # shellcheck disable=SC2206
+  # shellcheck disable=SC2206,SC2086
   run_and_redirect "${LOGDIR}/mvn_site.log" \
     "${MVN}" ${MVN_ARGS[*]} install \
       site site:stage \

